doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,71 @@ Tracing                                Jaeger
+| pas.mon.004 | The Monitoring service must at least perform active or passive        |                                                       |
+|             | monitoring                                                            |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.005 | The Monitoring service should be able to perform both active and      |                                                       |
+|             | passive monitoring                                                    |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.006 | The Monitoring service must support configuration of thresholds,      |                                                       |
+|             | outside of which the resource cannot function normally, for alert     |                                                       |
+|             | generation                                                            |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.007 | The Monitoring service must support configuration of alert            |                                                       |
+|             | notification medium (email, SMS, phone, etc.)                         |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.008 | The Monitoring service must support configurable re-alerting after    |                                                       |
+|             | a configurable period of time if the metric remains outside of the    |                                                       |
+|             | threshold                                                             |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.009 | The Monitoring service must support configurable alert escalations    |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.010 | The Monitoring service must support Alert acknowledgments by          |                                                       |
```

Comment:
Why "Alert" with capital "A" here, when it was using small "a" in other places?

Reply:
@walterkozlowski Thanks - done.